app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -236,7 +256,12 @@ class TopicRevisionFragmentTest {
   private fun clickRevisionTab() {
     onView(
       allOf(
-        withText(TopicTab.getTabForPosition(position = 3, enablePracticeTab).name),
+        withText(
+          TopicTab.getTabForPosition(
+            position = 3,
+            enablePracticeTab = enablePracticeTab
```

Comment:
added named argument, without it, it showing red marked error, nothing crashing or stopping app from working, but I think it's good to fix rather than keeping the red mark

Reply:
Ack.